src/js/cart.js

```diff
@@ -1,12 +1,14 @@
 import { getLocalStorage } from './utils.mjs';
+import { superscript } from './superscript.mjs'
 // import { } from '../../dist/images'; 
 
 function renderCartContents() {
   const cartItems = getLocalStorage('so-cart') || [];
   // Set an item = to 0 so you can add to it
   let subTotal = 0;
+  const itemsInCart = cartItems.length;
   // for loop just adds all Final Prices together in cartItems
-  for (let i = 0; i < cartItems.length; i++) {
+  for (let i = 0; i < itemsInCart; i++) {
```

Comment:
I changed this, it doesn't make a difference, but it is just a little easier to read this way personally. It does not have to stay this way though.